app/src/main/java/com/duckduckgo/app/browser/mediaplayback/MediaPlayback.kt

```diff
@@ -33,9 +33,8 @@ class RealMediaPlayback @Inject constructor(
     private val mediaPlaybackRepository: MediaPlaybackRepository,
 ) : MediaPlayback {
 
-    override fun isAutoplayEnabledForUrl(url: String): Boolean {
+    override fun doesMediaPlaybackRequireUserGestureForUrl(url: String): Boolean {
         val uri = url.toUri()
-        return mediaPlaybackFeature.self()
-            .isEnabled() && mediaPlaybackRepository.exceptions.firstOrNull { it.domain == uri.baseHost } != null
+        return !(mediaPlaybackFeature.self().isEnabled() && mediaPlaybackRepository.exceptions.any { it.domain == uri.baseHost })
```

Comment:
If `mediaPlaybackFeature.self().isEnabled()` then `doesMediaPlaybackRequireUserGestureForUrl` is false.

This still seems wrong to me.

Reply:
I was expecting:
```kotlin
mediaPlaybackFeature.self().isEnabled() && mediaPlaybackRepository.exceptions.firstOrNull { it.domain == uri.baseHost } != null
```

Reply:
@jonathanKingston This has been changed and now `webView.settings.mediaPlaybackRequiresUserGesture` is fully controlled by the `mediaPlaybackRequiresUserGesture` remote flag.